Fix for - 23979 (Update to find code in program.cs file in dot net 6 rather then start up ) (#24217)
dotnet/AspNetCore.Docs: aspnetcore/tutorials/first-web-api.md

* Enter *Web API* in the search box.
* Select the **ASP.NET Core Web API** template and select **Next**.
* In the **Configure your new project dialog**, name the project *TodoApi* and select **Next**.

> * In the **Additional information** dialog, confirm that **Framework** is **.NET 6.0 (Long-term support)**, checkbox **Use controllers(uncheck to use minimal APIs) is ticked** and select **Create**.

Reviewer: ```suggestion
* In the **Additional information** dialog:
  * Confirm the **Framework** is **.NET 6.0 (Long-term support)**.
  * Confirm the checkbox for **Use controllers(uncheck to use minimal APIs)** is checked.
  * Select **Create**.
```
